src/client/datascience/notebook/helpers/executionHelpers.ts

```diff
@@ -49,17 +49,16 @@ export function handleTensorBoardDisplayDataOutput(data: nbformat.IMimeBundle) {
  */
 export async function handleUpdateDisplayDataMessage(
     msg: KernelMessage.IUpdateDisplayDataMsg,
-    editor: NotebookEditor
+    document: NotebookDocument
 ): Promise<void> {
-    const document = editor.document;
     // Find any cells that have this same display_id
     for (const cell of document.cells) {
         if (cell.kind !== NotebookCellKind.Code) {
             continue;
         }
         let updated = false;
 
-        const outputs = createIOutputFromCellOutputs(cell.outputs);
+        const outputs = cell.outputs.map(translateCellDisplayOutput);
```

Comment:
Clean up, removed unnecessary function that does absolutely nothing but has a 1 line wrapper